question29/index.html

```diff
@@ -4,18 +4,51 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="./css/common.css" rel="stylesheet" />
     <link href="./css/news.css" rel="stylesheet" />
     <script>
       if (!localStorage.getItem("token")) window.location.href = "./login.html";
     </script>
-    <script src="./js/news.js" defer></script>
+    <script type="module" src="./js/news.js"></script>
+    <script type="module" src="./js/slideshow.js"></script>
     <title>TerraceTechフロントエンドエンジニア養成所</title>
   </head>
   <body>
+    <header>
+      <div class="header_container">
+        <ul class="header_nav">
+          <li class="header_nav_list">
+            <a href="#">
+              <img src="./img/account_icon.png" alt="アカウントアイコン画像">
+            </a>
+            <div class="header_nav_list_menu">
+              <ul>
+                <li class="menu">
+                  <a href="./reset-email.html">
+                      メールアドレス変更
+                  </a>
+                </li>
+                <li class="menu">
+                  <a href="./forgotPassword.html">
+                      パスワード変更
+                  </a>
+                </li>
+                <li class="menu">
+                  <a href="#" id="js-logout-btn">
+                      ログアウト
+                  </a>
```

Comment:
Q. Is it necessary to make it an a tag?

Reply:
I used the 'a' tag because it is easy to unify the style with other menus. However, since the 'a' tag is for page transitions, I thought its use was actually different. Originally, it was created with a button tag, so I modified to that tag. 

fixed commit : https://github.com/csakyo/homework/pull/48/commits/2008ed3dbb23a4a8057228593bddae2c1c2e4d17